test/e2e/suite/conformance/certificates/tests.go

```diff
@@ -212,6 +216,62 @@ func (s *Suite) Define() {
 			Expect(err).NotTo(HaveOccurred())
 		}, featureset.CommonNameFeature)
 
+		s.it(f, "should issue a basic, defaulted certificate for a single distinct DNS Name with specific RDN", func(issuerRef cmmeta.ObjectReference) {
```

Comment:
The current provided CAs don't support these feature. We can change the CAs we own and gate this test to them. (Not sure how to do that.)

Reply:
How many other CAs don't support this feature either? Does Vault, Let's Encrypt, Venafi etc support this? Is this something unlikely to be supported by most CAs out there?

Reply:
I think Vault should support this. LE probably doesn't because they have no way of validating the OUs (or other fields) and therefore don't allow them. I don't know about Venafi.